pyogrio/tests/test_geopandas_io.py

```diff
@@ -649,7 +706,7 @@ def test_write_dataframe_gdal_options_unknown(tmp_path, naturalearth_lowres):
     df = read_dataframe(naturalearth_lowres)
 
     # geojson has no spatial index, so passing keyword should raise
-    outfilename = tmp_path / "test.geojson"
+    outfilename = tmp_path / "test.geosjon"
```

Comment:
This seems a wrong change ;)
```suggestion
    outfilename = tmp_path / "test.geojson"
```

Reply:
Yeah thanks. sorry for the notification spam if you're getting it.